### ISSUE TITLE
Service  MD - Number of identificationInfo instances in a metadata record

### ISSUE THREAD

**Issue by ByronCinNZ** [icsm-au/metadata-working-group#2]
When creating a service metadata record, particularly one that is tightly coupled with the resource, one may wish to create multiple instances of identificationInfo instances (e.g. an SV_IndentificationInfo and a MD_DataIdentification) in a single metadata record. I think this is problematic and should be avoided as it means that a single metadata record could describe multiple resources. This could create much confusion when searching and even more when the data is harvested into catalogues that assume one resource per metadata record.

**Comment by MetadataToolsARDC** [comment]
I agree. We can decouple the metadata records (hence the one MD_DataIdentification or SV_ServiceIdentification object that each separate record contains) while meanwhile being able to represent tightly coupled services and datasets, which is important when not all operations of a service are compatible with the entire scope of dataset described by the relevant MD_DataIdentification.  Considering "Figure 18 - Service metadata information classes" in ANZ/NZS ISO 19115.1:2015 (ISO 19115-1:2014): for tightly coupled scenarios, this UML could be followed except for the following change:  instead of _operatesOn_ to a MD_DataIdentification object, we can consider _operatedDataset_ to a CI_Citation object.  Therefore also, SV_CoupledResource will have _resourceReference_ to the identical CI_Citation object for the dataset and no _resource_ (no link to an MD_DataIdentification object).  The CI_Citation would contain for example the DOI of the dataset which would often resolve to a catalogue web page for the metadata, but it will be important to be able to machine access the ISO19115-3 for the dataset from that CI_Citation content too (to discuss). This can ensure that sufficient SV_OperationMetadata will be provided as well as SV_CoupledResource with particular _scopedName_, so hopefully enough to satisfy use cases, including SV_Parameter and more.   It would be handy to be able to use xlink within the ISO19115-3 so that we would have one CI_Citation per dataset within the Service record (referenced from both SV_ServiceIdentification._operatedDataset_ and from SV_CoupledResource._resourceReference_) rather than having to maintain this information in both (or more) areas.
